analyzers/src/SonarAnalyzer.CFG/LiveVariableAnalysis/RoslynLiveVariableAnalysis.cs

```diff
@@ -138,6 +138,10 @@ private void BuildBranches(BasicBlock block)
                 AddPredecessorsOutsideRegion(finallyBlock);
             }
         }
+        if (block.IsEnclosedIn(ControlFlowRegionKind.Catch) && block.Successors.Any(x => x.Semantics == ControlFlowBranchSemantics.Rethrow))
```

Comment:
What about `Throw` semantics? All the UTs have `throw`, but `throw new Excetion("Whaaat?", ex)` should behave the same

Reply:
fixed here https://github.com/SonarSource/sonar-dotnet/pull/9530
Thanks @sebastien-marichal